src/middlewares/requestValidator/index.ts

```diff
@@ -5,12 +5,16 @@ import { configSchema } from './schema/config';
 export const requestValidator = (c: Context, next: any) => {
   const requestHeaders = Object.fromEntries(c.req.raw.headers);
 
+  const contentType = requestHeaders['content-type'];
   if (
-    requestHeaders['content-type'] &&
-    ![
+    !!contentType &&
+    (![
       CONTENT_TYPES.APPLICATION_JSON,
       CONTENT_TYPES.MULTIPART_FORM_DATA,
-    ].includes(requestHeaders['content-type'].split(';')[0])
+    ].includes(requestHeaders['content-type'].split(';')[0]) ||
```

Comment:
This nested OR condition looks incorrect. Its causing issues even when I send a valid content-type header